python/oneflow/nn/graph/block.py

```diff
@@ -526,6 +534,61 @@ def _shallow_repr(self):
         )
         return shallow_repr
 
+    def _ops_repr(self):
+        r"""Generate operators' string representation of this module
+        """
+        assert self._belonged_graph, "ModuleBlock: " + self._name_prefix + self.name + "'s belonged graph is not set."
+        ops_strs = []
+       
+        if self._belonged_graph.is_compiled:
+            module_conf = self._belonged_graph._full_graph_proto.module_name2module_conf[
```

Comment:
This can be get from _graph_proto ?

If get from _graph_proto, then when can debug this before the full graph is built.